rules/typescript.js

```diff
@@ -0,0 +1,62 @@
+const typescriptRules = {
+  '@typescript-eslint/adjacent-overload-signatures': 2,
+  '@typescript-eslint/array-type': 2,
```

Comment:
Should this enforce a kind? `Array<T>` or `T[]`?

Reply:
im might just delete this rule, actually, the default is `[2, 'array']` which makes `Array<thing>` an error